tools/python/run_CIs_for_external_pr.py

```diff
@@ -93,6 +93,11 @@ def main():
         # checks
         "onnxruntime-python-checks-ci-pipeline",
         "onnxruntime-binary-size-checks-ci-pipeline",
+
+        # not currently required, but running ensures we're hitting all mobile platforms
+        "Android CI Pipeline",
+        "iOS CI Pipeline",
+        "ONNX Runtime React Native CI Pipeline"
```

Comment:
Should we make these required CIs as well?

Reply:
They are still very unstable. Otherwise I agree with you we should do that.

Reply:
The pass rate for "ONNX Runtime React Native CI Pipeline" on main branch in last 14 days was just 56%.

Reply:
Since the Android emulator setup was updated it has been much better. Recent failures seem to be due to Component Detection timeouts.